packages/client/src/generation/TSClient/PrismaClient.ts

```diff
@@ -307,6 +307,22 @@ function runCommandRawDefinition(this: PrismaClientClass) {
   return ts.stringify(method, { indentLevel: 1, newLine: 'leading' })
 }
 
+function applyPendingMigrationsDefinition(this: PrismaClientClass) {
```

Comment:
See suggestion above about migrations being automatically applied on startup rather than requiring explicit `applyPendingMigrations` call from the user. Rest of the comments on this function apply only if we can't/do not want to do this.